test/Test/Database/LSMTree/Internal/BloomFilter.hs

```diff
@@ -205,3 +207,18 @@ instance Arbitrary Bloom2.CandidateProbe where
 
 deriving stock instance Eq Bloom2.CandidateProbe
 #endif
+
+prop_bloomInserts :: FPR -> [Word64] -> Property
+prop_bloomInserts (FPR fpr) keys =
+     bloom_insert === bloom_insertMany
+  where
+    bloom_insert =
+      BF.create (BF.sizeForFPR fpr n) $ \mb ->
+        mapM_ (BF.insert mb) keys
+
+    bloom_insertMany =
+      BF.create (BF.sizeForFPR fpr n) $ \mb ->
+        Bloom.bloomInserts mb (\k -> pure $! keysv V.! k) n
+
+    !keysv = V.fromList keys
+    !n     = V.length keysv
```

Comment:
It might be more fitting to put this test into the bloomfilter test suit instead of the lsm-tree test suite

Reply:
If so, we'd want to move the bulk insert function into the `bloomfilter` lib too.

Reply:
Oops, yes, true, I was under the impression that it was already in the `bloomfilter` lib. I don't have a preference, so do what you see fit to do

Reply:
I was also thinking I might move it.

The only thing is, then we'd want a corresponding multi-lookup. But the lookup we want is for N * M keys and filters, which would not reuse a multi-lookup for many keys in one filter, since we want a sparse result array.